tests/test_other.py

```diff
@@ -9938,10 +9938,9 @@ def test(args, expect_fail):
     # but work with it
     self.assertContained('hello, world!', test(['-s', 'LEGACY_VM_SUPPORT'], expect_fail=False))
 
-  # Compile-test for -s USE_WEBGPU=1 and library_webgpu.js.
-  def test_webgpu_compiletest(self):
+  def test_webgpu_jsvalstore(self):
```

Comment:
Isn't it true that this is still a compile test?  i.e. the fact that the file we use here is `webgpu_jsvalstore.cpp` is not important is it?

Reply:
That's true yes, I guess I was thinking closure would provide some verification of the contents of that test too, but we don't run this with closure.

Reply:
True, we probably should be building with closure here then too!  

I was thinking it is logically here so that we can have something that can be tested without a browser so maybe it makes sense to call it a compile test?

Reply:
Agree. I've renamed the test back - and will see if I can test with closure too, as that's actually very useful. (Though there is fortunately another test which is testing all of the `library_*.js` with closure too; it caught several errors in this PR earlier.)

Reply:
Added that coverage and updated the externs to make closure happy with the EM_ASM code in that test. Thanks for your review!